mmv1/compile/openapi_parser.rb

```diff
@@ -0,0 +1,153 @@
+require 'openapi_parser'
+
+def writeObject(name, obj, type, url_param)
+  res = nil
+  case type
+  when "string"
+    res = Api::Type::String.new(name)
+  when "integer"
+    res = Api::Type::Integer.new(name)
+  when "number"
+    res = Api::Type::Double.new(name)
+  when "boolean"
+    res = Api::Type::Boolean.new()
+    res.instance_variable_set(:@name, name)
```

Comment:
And string is different because it has a constructor already. Thanks!

I'd like to eliminate that or use it for the others too by moving it up to their parent type- but that's not a yak to shave in this PR.